draft-ietf-quic-transport.md

```diff
@@ -3700,7 +3700,9 @@ when it receives its first Handshake packet.  Though packets might still be in
 flight or awaiting acknowledgment, no further Initial packets need to be
 exchanged beyond this point.  Initial packet protection keys are discarded (see
 Section 4.10 of {{QUIC-TLS}}) along with any loss recovery and congestion
-control state (see Sections 5.3.1.2 and 6.9 of {{QUIC-RECOVERY}}).
+control state (see Sections 5.3.1.2 and 6.9 of {{QUIC-RECOVERY}}). This is safe
+because of the additional safeguards against loss of Handshake packets in
```

Comment:
The reason Initial keys can be discarded is because the endpoint knows the peer has Handshake keys, so this new statement is confusing to me.

Reply:
The peer having Handshake keys is not sufficient. The peer also needs to ensure that packets are repeated.

Reply:
The packets need to be repeated to avoid deadlock and to ensure the handshake completes, but not in order to be sure that the CRYPTO data sent in Initial has arrived.

I suspect I'm missing something here?